### ISSUE TITLE
Build system

### ISSUE THREAD

**Issue by rapodaca** [rapodaca/dialect#4]
It should be possible to build a preprint-ready manuscript in PDF format. The system should use [Markdown](https://daringfireball.net/projects/markdown/) as the source for accessibility.

Some ideas:

- [Boilerplating Pandoc for Academic Writing](https://www.soimort.org/notes/161117/)
- [Scientific Writing with Markdown](https://jaantollander.com/post/scientific-writing-with-markdown/)
- [Writing a scientific paper in Markdown](https://github.com/mhyfritz/scientific-paper-markdown-template)
- [Why and How I Write Scientific Documents in Plain Text](https://economicsfromthetopdown.com/2020/12/10/why-and-how-i-write-scientific-documents-in-plain-text/)
